.github/workflows/gh-pages.yml

```diff
@@ -8,6 +8,9 @@ jobs:
   publish:
     name: Publish Storybook to GitHub Pages
     runs-on: ubuntu-20.04
+    permissions:
+      contents: read
+      packages: write
```

Comment:
This particular action probably doesn't need to write to `packages`. But I think the other workflow, `gh-packages.yml` will